src/theme/default_theme.rs

```diff
@@ -104,6 +104,7 @@ impl UiStyles {
                 temp:       White.normal(),
                 compiled:   Yellow.normal(),
                 build:      Yellow.bold().underline(),
+                source:     Yellow.bold(), // Need to discuss color
```

Comment:
I think yellow bold is already somewhere, but if not its good if people are not happy they will change it

Reply:
Yeah I just saw the problem is it is to near the build one

Reply:
That was sort of intentional.  Build scripts/config or a sort of source code also.

Reply:
Yeah but I think that even tho they are sort of source code and are useful all together, but I think that if @cafkafk is okay with that we just keep it and I will change my eza colors